package.json

```diff
@@ -79,7 +79,8 @@
     "request": "2.83.0",
     "sinon": "4.4.2",
     "tar": "4.4.0",
-    "webpack": "3.10.0",
+    "webpack": "4.0.1",
```

Comment:
It seems like it was actually `webpack` 3.x that was used before, `webpack-grunt` doesn't have webpack 4.x release supported yet (it's merged into master though so should only be a matter of time 🎉)

I also addressed this in #1889, but maybe this has all the changes required to go to webpack 4.x?

Reply:
Just merged or going to merge the removal of `webpack-grunt` dependency so we'll be able to update to webpack 4.x soon. But step by step…

I'll back out the package.json changes here anyway.